README.md

```diff
@@ -55,3 +62,15 @@ try {
 
 ```
 
+## Limitations
+
+- Non-ember npm dependencies are not supported.  It is assumed all resolutions one of:
+   - classic ember apps
+   - classic ember addons
+   - v2 ember addons
```

Comment:
Would you mind making an issue to track this? I think we do need to plan to support this (especially as we move forward with Embroider, "plain npm" packages are going to be much more common even for Ember app specific things).

Reply:
#8